### DISCUSSION TITLE
54.144.189.255 frontend.sh(error)

### DISCUSSION THREAD

**Discussion by sravanreddy28** [learndevopsonline/learndevopsonline#1235] [Q&A]
Hi @r-devops can you please help me on this issue facing from 2 days uanble to sortout while executing the script!!!

image : https://github.com/learndevopsonline/learndevopsonline/assets/138639644/f54e5be4-c458-4082-9f7a-0f233d1930f1

Getting error like : **cp: cannot create regular file '/etc/nginx/dafault.d/roboshop.conf': No such file or directory**

**Comment by maheshbusireddy** [comment]
@sravanreddy28 There is a typo error in your frontend.sh file. its not /etc/nginx/dafault.d/roboshop.conf it is like /etc/nginx/default.d/roboshop.conf. Modify the code and re run

**Reply by sravanreddy28** [reply]
yeh got it @maheshbusireddy

Thanks,

**Reply by Tirupathi1729** [reply]
bro its running correctly now

**Reply by sravanreddy28** [reply]
yeh working @Tirupathi1729  but categories are not displaying

**Reply by sravanreddy28** [reply]
let me check once @Tirupathi1729

**Reply by sravanreddy28** [reply]
@Tirupathi1729  (**54.208.117.51 (mongo ip address)** mongodb ip address is good its alredy changed from 127.0.0.1 to 0.0.0.0 and in nginx record name is also good

image: https://github.com/learndevopsonline/learndevopsonline/assets/138639644/8ae0e7ad-532a-465e-ad19-fa9545bce5ab
**54.90.123.227(catalogue IP)**

**Reply by sravanreddy28** [reply]
let check @Tirupathi1729

**Reply by sravanreddy28** [reply]
@Tirupathi1729  image : 
https://github.com/learndevopsonline/learndevopsonline/assets/138639644/ccb6faca-2f88-4e01-a899-08ce82785cc8

its good already given mongodIP

**Reply by sravanreddy28** [reply]
@Tirupathi1729 (**54.90.123.227(catalogue IP)**

**Comment by Tirupathi1729** [comment]


**Reply by sravanreddy28** [reply]
@Tirupathi1729 **(54.90.123.227(catalogue IP)**
